tests/integration/API/MP/APITest.php

```diff
@@ -154,191 +98,21 @@ public function testItUsesMultipleListsSubscribeMethodWhenSubscribingToSingleLis
-    expect($result['id'])->equals($subscriber->id);
-    expect($result['subscriptions'])->notEmpty();
-    expect($result['subscriptions'][0]['segment_id'])->equals($segment->id);
-  }
-
-  public function testItSchedulesWelcomeNotificationByDefaultAfterSubscriberSubscriberToLists() {
-    $API = Stub::makeEmptyExcept(
-      \MailPoet\API\MP\v1\API::class,
-      'subscribeToLists',
-      [
-        '_scheduleWelcomeNotification' => Expected::once(),
-        'settings' => SettingsController::getInstance(),
-      ], $this);
-    $subscriber = Subscriber::create();
-    $subscriber->hydrate(Fixtures::get('subscriber_template'));
-    $subscriber->status = Subscriber::STATUS_SUBSCRIBED;
-    $subscriber->save();
-    $segment = Segment::createOrUpdate(
-      [
-        'name' => 'Default',
-        'type' => Segment::TYPE_DEFAULT,
-      ]
-    );
-    $API->subscribeToLists($subscriber->id, [$segment->id], ['skip_subscriber_notification' => true]);
-  }
-
-  public function testItDoesNotScheduleWelcomeNotificationAfterSubscribingSubscriberToListsWhenDisabledByOption() {
-    $API = Stub::makeEmptyExcept(
-      \MailPoet\API\MP\v1\API::class,
-      'subscribeToLists',
-      [
-        '_scheduleWelcomeNotification' => Expected::never(),
-        'settings' => SettingsController::getInstance(),
-      ], $this);
-    $subscriber = Subscriber::create();
-    $subscriber->hydrate(Fixtures::get('subscriber_template'));
-    $subscriber->status = Subscriber::STATUS_SUBSCRIBED;
-    $subscriber->save();
-    $segment = Segment::createOrUpdate(
-      [
-        'name' => 'Default',
-        'type' => Segment::TYPE_DEFAULT,
-      ]
-    );
-    $options = ['schedule_welcome_email' => false, 'skip_subscriber_notification' => true];
-    $API->subscribeToLists($subscriber->id, [$segment->id], $options);
-  }
-
   public function testItGetsSegments() {
-    $segment = Segment::createOrUpdate(
-      [
-        'name' => 'Default',
-        'type' => Segment::TYPE_DEFAULT,
-      ]
-    );
+    $segment = $this->getSegment();
     $result = $this->getApi()->getLists();
     expect($result)->count(1);
-    expect($result[0]['id'])->equals($segment->id);
+    expect($result[0]['id'])->equals($segment->getId());
   }
 
   public function testItExcludesWPUsersAndWooCommerceCustomersSegmentsWhenGettingSegments() {
-    $defaultSegment = Segment::createOrUpdate(
-      [
-        'name' => 'Default',
-        'type' => Segment::TYPE_DEFAULT,
-      ]
-    );
-    $wpSegment = Segment::createOrUpdate(
-      [
-        'name' => 'Default',
-        'type' => Segment::TYPE_WP_USERS,
-      ]
-    );
-    $wcSegment = Segment::createOrUpdate(
-      [
-        'name' => 'Default',
-        'type' => Segment::TYPE_WC_USERS,
-      ]
-    );
+    $defaultSegment = $this->getSegment();
+    $this->getSegment('WordPress', SegmentEntity::TYPE_WP_USERS);
+    $this->getSegment('WooCommerce', SegmentEntity::TYPE_WC_USERS);
```

Comment:
I have changed the names of the segments to be different because of [MAILPOET-3998]